sections/header.liquid

```diff
@@ -615,6 +666,9 @@
   }
 
   customElements.define('sticky-header', StickyHeader);
+
+  const header = document.querySelector('header');
+  if (header) document.documentElement.style.setProperty('--header-bottom-position', `${Math.floor(header.getBoundingClientRect().bottom)}px`);
```

Comment:
I think Martina had brought up that maybe this could run only if mega menu is enable ? 🤔

Reply:
Updated https://github.com/Shopify/dawn/pull/1316/commits/2110cf31f8c9dafa6485485fa7ea6cc33c321884